packages/app/src/runner/index.ts

```diff
@@ -283,17 +309,30 @@ function runSpecE2E (spec: SpecFile) {
 
   const $autIframe: JQuery<HTMLIFrameElement> = autIframe.create().appendTo($container)
 
+  // Remove the spec bridge iframe
+  document.querySelectorAll('iframe.spec-bridge-iframe').forEach((el) => {
```

Comment:
slight mutation from [original](https://github.com/cypress-io/cypress/pull/18075/files#diff-bd8f567e7cefe6212c99b7760b613f12ffa518138f1d19aff5f3e5cf0c677bf9R161) as we cannot leverage `Cypress.$` in this scenario (at least from what I was seeing at the time)